elasticsearch5/src/test/java/org/vertexium/elasticsearch5/Elasticsearch5SearchIndexTest.java

```diff
@@ -175,4 +176,9 @@ private Throwable getRootCause(Throwable e) {
         }
         return getRootCause(e.getCause());
     }
+
+    @Override
+    protected ScoringStrategy getHammingDistanceScoringStrategy(String field, String hash) {
+        return new ElasticsearchHammingDistanceScoringStrategy(field, hash);
```

Comment:
I would love to avoid having to create scoring strategies specific to ES but I can't think of a way around that.